### DISCUSSION TITLE
Unable to Sync Changes/Commit

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#187] [Q&A] [answered]
When I try to commit and sync changes in source control I receive this message and no changes are committed, does anyone know why I am receiving this error?
![Screen Shot 2023-07-11 at 5 07 05 PM](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133035062/0bdd01e0-7d48-4091-8808-735c97e0e1b2)

**Comment by jakegros** [comment]
Did you try to "Open in Git Log"?
You can then see the output running and check for any problems.  It is one of the tabs.

**Reply by jakegros** [reply]
After a while, when I sync my code it starts taking quite a long time.  Either that or I have to keep trying to deploy.
Here are some [Intro videos for VS Code](https://code.visualstudio.com/docs/getstarted/introvideos) that I think are helpful.

**Comment by derekscholes** [comment]
Sometimes if you make your codespace and afterwards you add something to your repo, it does this because it changes the source folder you are editing. You may have to create a new codespace if you made any changes to your files outside of codespaces.

**Comment by audreyhki** [comment, marked as answer]
I met with Professor Synder and found the solution to my problem. I was able to "commit" changes in codespaces but was given this error code whenever I tried to "sync changes". By opening the Git Log when this pop-up came up, I was able to copy a suggested line of code from the Git Log that merged the multiple branches I had. I pasted it into the Terminal, and after saving, I was once again able to "sync changes".
